projects/libs/flex-layout/package.json

```diff
@@ -29,7 +29,7 @@
     "tslib": "^2.3.0"
   },
   "peerDependencies": {
-    "@angular/cdk": "^13.0.0",
+    "@angular/cdk": "^14.0.0",
```

Comment:
Would it be possible to use peer deps similar to how Angular Material does, which includes the next major version as well, to ease the update process.  e.g. `^14.0.0 || ^15.0.0`.  This should be safe and would improve the DX for this interim period between angular and flex layout releases.